blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubOrganizationFolder.java

```diff
@@ -32,10 +34,12 @@ public boolean isScanAllRepos() {
             SCMNavigator scmNavigator = getFolder().getSCMNavigators().get(0);
             if(scmNavigator instanceof GitHubSCMNavigator){
                 GitHubSCMNavigator gitHubSCMNavigator = (GitHubSCMNavigator) scmNavigator;
-                return (StringUtils.isBlank(gitHubSCMNavigator.getIncludes()) || gitHubSCMNavigator.getIncludes().equals("*"))
-                        && StringUtils.isBlank(gitHubSCMNavigator.getExcludes())
-                        && (StringUtils.isBlank(gitHubSCMNavigator.getPattern())
-                        || gitHubSCMNavigator.getPattern().equals(".*"));
+                for (SCMTrait<?> t: gitHubSCMNavigator.getTraits()) {
+                    if (t.getDescriptor().getClass().getAnnotation(Selection.class) != null) {
```

Comment:
@vivek unclear what the intent of this getter is... are you checking if there are any restrictions at all in place (in which case this is correct)... or are you checking if blueocean is managing the repository selection (in which case we just want 

```
if (t instanceof BlueOceanSCMSourceFilterTrait) {
  return false;
}
```